services/spar/test-integration/Test/Spar/SCIMSpec.hs

```diff
@@ -115,12 +116,57 @@ randomUser = do
         , SCIM.User.externalId  = Just ("scimuser_extid_" <> suffix)
         }
 
+specTokens :: SpecWith TestEnv
+specTokens = describe "operations with provisioning tokens" $ do
+    describe "POST /auth-tokens" $ do
+        it "creates a usable token" $ do
+            env <- ask
+            -- Create a token
+            token <- createToken CreateScimToken
+                { createScimTokenDescription = "token creation test" }
+            -- Try to execute a SCIM operation without a token and check
+            -- that it fails
+            listUsers_ Nothing Nothing (env ^. teSpar)
+                !!! const 401 === statusCode
+            -- Try to execute the same SCIM operation with the generated
+            -- token; it should succeed now
+            listUsers_ (Just token) Nothing (env ^. teSpar)
+                !!! const 200 === statusCode
+            -- Cleanup
+            deleteToken token
+        it "respects the token limit (2 for integration tests)" $ do
```

Comment:
If it is only ok to have 1 token, but not 2, then the limit should be `1`, no?

Reply:
The limit is "maxScimTokens is allowed, maxScimTokens+1 is not allowed". One token is already present because we created it in TeamEnv. I'll add a comment clarifying that,